.github/workflows/nuget-package-tests.yml

```diff
@@ -15,35 +15,36 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [windows-latest, ubuntu-20.04, macos-latest]
     steps:
       - uses: actions/checkout@v2
-      - name: Setup .NET Core 5 (On Mac)
-        if: ${{ matrix.os == 'macos-latest' }}
-        uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.100
-        env:
-          DOTNET_INSTALL_DIR: /users/runner/alternate/dotnet
-      - name: Setup .NET Core 5 (Not on mac)
-        if: ${{ matrix.os != 'macos-latest' }}
+          submodules: true
+          fetch-depth: 0
+      - name: Setup .NET Core
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 5.0.100
-      - name: Download Drivers
-        run: dotnet run -p ./src/tools/Playwright.Tooling/Playwright.Tooling.csproj -- download-drivers --basepath .
-      - name: Build Project
-        run: dotnet build ./src/Playwright/Playwright.csproj -c Debug
-      - name: Download Browsers
-        run: dotnet run -p ./src/Playwright -f net5.0 install
-      - name: Delete browsers on Mac
-        if: matrix.os == 'macos-latest'
-        run: rm -r ~/Library/Caches/ms-playwright
-      - name: Delete browsers on Linux
-        if: ${{ matrix.os == 'ubuntu-latest' }}
-        run: rm -r ~/.cache/ms-playwright
-      - name: Delete browsers on windows
-        if: matrix.os == 'windows-latest'
-        run: Remove-Item -path $env:USERPROFILE\AppData\Local\ms-playwright -recurse
+          dotnet-version: 5.0.x
+      - name: Install prerequisites and download drivers
+        shell: pwsh
+        run: .\build.ps1 driver -prereqs
+      - name: Install Playwright CLI
+        run: dotnet tool install --global Microsoft.Playwright.CLI --version 1.0.1-next-1
```

Comment:
maybe not hardcode the version there? but unsure.

Reply:
So, this ~is~ was there because CLI wasn't pushed until yesterday, and the tool will default to the release version which failed to correctly find the DLLs needed in this case.